app/src/main/java/ch/heigvd/pro/b04/android/Home/HomeViewModel.java

```diff
@@ -44,7 +78,10 @@ public void addNewEmoji(Emoji emoji) {
             buffer.clear();
             registrationCode.postValue(code.toString());
 
-            // Use retrofit -> https://square.github.io/retrofit/
+            RetrofitClient.getRetrofitInstance()
+                    .create(RockinAPI.class)
+                    .postConnect(new SessionCode(code.toString()))
+                    .enqueue(callbackToken);
```

Comment:
In the future, we might want to have an application-scoped RockinAPI instance instead.